File: benchmarkDS/Vanilla/GEMINI_gemini-2.5-pro/Lse11.py
import sys

def getValueFromArray(arr, size, index):
  return arr[index]

if __name__ == "__main__":
    id_sequence = [10, 20, 30, 40, 50]
    size = len(id_sequence)

    # 5 Test cases using assertions
    assert getValueFromArray(id_sequence, size, 0) == 10
    assert getValueFromArray(id_sequence, size, 1) == 20
    assert getValueFromArray(id_sequence, size, 2) == 30
    assert getValueFromArray(id_sequence, size, 3) == 40
    assert getValueFromArray(id_sequence, size, 4) == 50

    try:
        index_str = input()
        if index_str:
            index = int(index_str)
            value = getValueFromArray(id_sequence, size, index)
            sys.exit(value)
    except (ValueError, IndexError):
        sys.exit(1) # Exit with an error code for bad input or out-of-bounds

File: benchmarkDS/Vanilla/GEMINI_gemini-2.5-pro/Lse11.c
#include <stdio.h>
#include <assert.h>

int getValueFromArray(int arr[], int size, int index) {
    return arr[index];
}

int main() {
    int id_sequence[] = {10, 20, 30, 40, 50};
    int size = sizeof(id_sequence) / sizeof(id_sequence[0]);
    
    // 5 Test cases using assertions
    assert(getValueFromArray(id_sequence, size, 0) == 10);
    assert(getValueFromArray(id_sequence, size, 1) == 20);
    assert(getValueFromArray(id_sequence, size, 2) == 30);
    assert(getValueFromArray(id_sequence, size, 3) == 40);
    assert(getValueFromArray(id_sequence, size, 4) == 50);

    int index;
    if (scanf("%d", &index) == 1) {
        if (index >= 0 && index < size) {
            int value = getValueFromArray(id_sequence, size, index);
            return value;
        } else {
            return 1; // Return an error code for out-of-bounds index
        }
    }
    return 1; // Return an error code for non-integer input
}